spec/rubocop/runner_spec.rb

```diff
@@ -60,6 +60,40 @@ def INVALID_CODE
         END
       end
     end
+
+    context 'if -s/--stdin is used with an offense' do
+      let(:options) do
+        {
+          formatters: [['progress', formatter_output_path]],
+          stdin: <<-END.strip_indent
```

Comment:
Looking at your other spec I was under the impression you need to pass some path as well.